### ISSUE TITLE
github.api methods inside a plugin

### ISSUE THREAD

**Issue by jamtur01** [danger/danger#319]
I'm trying to use the `github.api.` methods inside a plugin:

```ruby
....
          f.chomp if github.api.organization_member?('kickstarter', f)
...
```

I get a method missing error:

```
undefined method `github' for #<Danger::Dangerfile:0x007faf43ba0a00> (NoMethodError)
	/Users/james/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/bundler/gems/danger-7b53d32348a7/lib/danger/danger_core/dangerfile.rb:64:in `method_missing'
	/Users/james/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/bundler/gems/danger-7b53d32348a7/lib/danger/plugin_support/plugin.rb:21:in `method_missing'
	/Users/james/src_work/kickstarter/danger_plugins/danger_blame.rb:8:in `block (2 levels) in who_to_blame'
	/Users/james/src_work/kickstarter/danger_plugins/danger_blame.rb:6:in `each'
	/Users/james/src_work/kickstarter/danger_plugins/danger_blame.rb:6:in `block in who_to_blame'
	/Users/james/src_work/kickstarter/danger_plugins/danger_blame.rb:5:in `each'
	/Users/james/src_work/kickstarter/danger_plugins/danger_blame.rb:5:in `who_to_blame'
	/Users/james/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/bundler/gems/danger-7b53d32348a7/lib/danger/danger_core/dangerfile.rb:116:in `block (2 levels) in method_values_for_plugin_hashes'
	/Users/james/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/bundler/gems/danger-7b53d32348a7/lib/danger/danger_core/dangerfile.rb:115:in `map'
	/Users/james/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/bundler/gems/danger-7b53d32348a7/lib/danger/danger_core/dangerfile.rb:115:in `block in method_values_for_plugin_hashes'
	/Users/james/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/bundler/gems/danger-7b53d32348a7/lib/danger/danger_core/dangerfile.rb:111:in `each'
	/Users/james/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/bundler/gems/danger-7b53d32348a7/lib/danger/danger_core/dangerfile.rb:111:in `flat_map'
	/Users/james/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/bundler/gems/danger-7b53d32348a7/lib/danger/danger_core/dangerfile.rb:111:in `method_values_for_plugin_hashes'
	/Users/james/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/bundler/gems/danger-7b53d32348a7/lib/danger/danger_core/dangerfile.rb:132:in `print_known_info'
	/Users/james/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/bundler/gems/danger-7b53d32348a7/lib/danger/danger_core/dangerfile.rb:154:in `parse'
	/Users/james/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/bundler/gems/danger-7b53d32348a7/lib/danger/commands/local.rb:66:in `run'
```

**Comment by jamtur01** [comment]
Seems fixed in 0.8.5.

**Comment by orta** [comment]
👍
